<launch>
  <node pkg="win_camera" type="win_camera_node" name="win_camera_node">
    <param name="frame_id" value="WinCamera1" />
    <param name="image_width" value="1280" />
    <param name="image_height" value="720" />
    <param name="frame_rate" value="30.0" />
  </node>
</launch>
